scopes/scope/ui/empty-scope/empty-scope.module.scss

```diff
@@ -0,0 +1,4 @@
+.text {
+  max-width: 400px;
```

Comment:
max-width is difficult to work with!
better to have **regular content with padding**, i.e. no width property and set **`display: flex` on parent**.